components/gpio_control/example_configs/phoniebox_gpio_control.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Phoniebox GPIO Control Service
+After=network.target iptables.service firewalld.service
+
+[Service]
+Type=simple
+User=pi
+Group=pi
+WorkingDirectory=/home/pi/RPi-Jukebox-RFID/components/gpio_control/
+ExecStart=/home/pi/RPi-Jukebox-RFID/components/gpio_control/gpio_control.py
```

Comment:
At the moment we prefix the python file with `/usr/bin/python3` in all service definitions to make sure Python 3 is used

Reply:
@s-martin : I do not fully understand what you want to highlight with this comment. 
Instead of using an explizit path here i would suggest to use the shebang line 
`#!/usr/bin/python3`

this would also allow later to use a virtual environment, as this would be automatically be changed if we find a way to 'properly' install the service

Reply:
actually another question i have is how we should work with paths in the future...
It feels wrong to have the explicit path assuming an installation into /home/pi/RPi-Jukebox-RFID/

Reply:
We had the explicit path to the executable beginning of this year as a hotfix for some immediate bugs.

I agree that the correct way would be to use the shebang line, so please ignore my comment :)

I thought about the absolute paths, too, but right now I don't have a good idea how we should handle this and what would be the way to migrate existing installations.